src/main/java/com/exasol/errorcodecrawlermavenplugin/ErrorCodeCrawlerMojo.java

```diff
@@ -38,15 +39,29 @@
     @Parameter(name = "excludes")
     private List<String> excludes;// this variable must have the same name as the parameter
 
+    @Parameter(name = "sourcePaths")
+    private List<String> sourcePaths;
+
+    // [impl->dsn~src-directrories]
+    // [impl->dsn~src-directrory-override]
+    private List<Path> getSourcePaths() {
+        if (this.sourcePaths == null || this.sourcePaths.isEmpty()) {
+            return List.of(Path.of("src", "main", "java"), Path.of("src", "test", "java"));
+        } else {
+            final String separator = System.getProperty("file.separator");
+            return this.sourcePaths.stream().map(string -> Path.of(string.replace("/", separator)))
```

Comment:
Would it work on windows?

Reply:
Yes. I intentionally enforce the `/` here since in the pom.xml configuration it should also be the slash on windows.
That's why I replace it here by an OS specific sepatator.